<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEncryptor</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Шифрование</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Расшифровка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>390</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Зашифровать в файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Расшифровать файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onCryptClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onDecryptClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onExitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCryptClick()</slot>
  <slot>onDecryptClick()</slot>
  <slot>onExitClick()</slot>
 </slots>
</ui>
